cognite/powerops/client/shop/cogshop_api.py

```diff
@@ -201,16 +202,17 @@ def retrieve_shop_result(self, result_external_id: str) -> ShopResult:
         return self._po.shop_based_day_ahead_bid_process.shop_result.retrieve(external_id=result_external_id)
 
     def list_shop_versions(self) -> list[str]:
-        """List the available version of SHOP remotely  in CDF.
-        Does not include versions in local version of CShaaS.
+        """List the available version of SHOP in CDF. Does not include versions that are "local" to CShaaS.
         SHOP releases should have the following format:
-        'SHOP-{VERSION}-pyshop-python{py_version}.linux.zip'
+        'SHOP-{VERSION}-pyshop-python{py_version}.linux.zip' and the metadata {'shop:type': 'shop-release'}.
         """
         # todo? Add an endpoint to list the available versions of SHOP via powerops API?
         return [file.name for file in self._cdf.files.list(metadata={"shop:type": "shop-release"}, limit=-1)]
 
     def retrieve_shop_case_graphql(self, case_external_id: str) -> ShopCase:
-        """Retrieve a shop case from CDF using GraphQL. This is a a way to retrieve a ShopCase instance."""
+        """Retrieve a shop case from CDF using GraphQL.
+        Instead of the external of IDs of ShopScenario, ShopModel and ShopFiles, the actual instances are fetched.
+        This is more stable than retrieving a ShopCase using `retrieve_shop_case` but less efficient."""

```

Comment:
I guess that's the truth, but do we really want to highlight that the other method is unstable?